test/AutomatedCarTests/SystemComponents/Packets/DebugPacketTests.cs

```diff
@@ -30,6 +36,27 @@ public void RadarSensorExsits()
             Assert.IsType<bool>(debugPacket.RadarSensor);
         }
 
+        [Fact]
+        public void WithNoInputPolygonSetDoesentChangeTheStateOfPolygon()
+        {
+            var polygon = debugPacket.Polygon;
+            debugPacket.PolygonSet(false);
+
+            Assert.Equal(polygon, debugPacket.Polygon);
+        }
+
+        [Fact]
+        public void WithNoumerusInputPolygonSetSwitchBetweenOnAndOff()
+        {
+            var polygon = debugPacket.Polygon;
+            for (int i = 0; i < 10; i++)
+            {
+                Assert.Equal(polygon, debugPacket.Polygon);
+                debugPacket.PolygonSet(true);
+                polygon = !polygon;
+            }
+        }
+
         [Fact]
         public void WithNoInputUtrasoundSensorSetDoesentChangeTheStateOfUtrasoundSensor()
```

Comment:
DoesNot vagy Doesnt nem tudom melyik a jobb, de az e nem kell.
Viszont a metódusnevek kezdődjenek igével (felszólító mód), akkor is ha teszt. Ez a másikra is igaz.